code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -18,6 +18,8 @@
 		/mob/living/simple_animal/pet/dog/corgi,\
 		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
 		/mob/living/simple_animal/bot/ed209,\
+		/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper,\
+		/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife,\
```

Comment:
PLAYER MIDWIFE nurse spiders? There is NO WAY this can end badly! :smiling_imp: :

Reply:
<3